ember-cli-build.js

```diff
@@ -4,10 +4,21 @@ var EmberApp = require('ember-cli/lib/broccoli/ember-app');
 var environment = EmberApp.env();
 var minifyEnabled = environment === "production" || environment === "staging";
 
+var babel = {
+  plugins: []
+};
+if (environment === 'development') {
+  // Add CodeSee instrumentation in development mode
+  babel.plugins.push([
+    require.resolve("@codesee/babel-plugin-instrument"), { frameworks: ["ember"]}
```

Comment:
@jleven if you can ignore the eslint check for the require.resolve, the test would pass. 
The error generated currently is 

`Error:   13:21  error  "@codesee/babel-plugin-instrument" is extraneous  node/no-extraneous-require`

Reply:
Thanks @subho007! It looks like we don't need `require.resolve` here at all, so I've just removed it. I also resolved the merge conflict and cleaned up the history while I was at it. Fingers crossed, this should be good to merge now 🤞